etn-ccis/blui-react-workflows
# Feature/4592 browser back workflow first screen

Fixes # BLUI-4592



#### Changes proposed in this Pull Request:

- Added navigate browser back on the first screen of registration workflow
- Refactored children of registration workflow logic
- Updated test cases



#### Screenshots / Screen Recording (if applicable)

https://github.com/etn-ccis/blui-react-workflows/assets/120575281/c34c480e-9210-49a7-9dbf-1bb556d7175b



#### To Test:
- cd login
- yarn start:example
- Click on Create Account button
- Click on Back button on the first screen
- It should navigate to previous page in the browser



#### Any specific feedback you are looking for?

-
